cmd/lk/app.go

```diff
@@ -87,18 +87,12 @@ var (
 						},
 						&cli.StringFlag{
 							// Use "http://cloud-api.livekit.run" in local dev
+							// Use "https://cloud-api-server-staging.ochicago1a.staging.livekit.app" in staging
```

Comment:
any way to keep these out of the OSS repo?

Reply:
We can remove it, just there as a courtesy 👍